rocAL/rocAL_pybind/examples/tf_petsTrainingExample/requirements.sh

```diff
@@ -0,0 +1,8 @@
+# installing required packages
+pip install contextlib2
```

Comment:
Are these changes needed in this PR?

Reply:
A change from ToT not reflected in Shobana's fork